packages/http/src/lib.ts

```diff
@@ -31,7 +31,7 @@ export const $lib = createTypeSpecLibrary({
     "optional-path-param": {
```

Comment:
```suggestion
    "optional-needs-path-expansion": {
```
think that will be more accurate